web/js/naturalEvents/wv.naturalEvents.model.js

```diff
@@ -42,25 +25,29 @@ wv.naturalEvents.model = wv.naturalEvents.model || function(models, config) {
 
   self.save = function(state) {
     if (self.active) {
-      state.e = 't';
+      var id = wvx.ui.naturalEvents.selected.id;
+      var date = wvx.ui.naturalEvents.selected.date;
+      var value = id?date?[id, date].join(','):id:true;
+      state.e = value;
     }
   };
 
   self.load = function(state) {
-    if (state.e == 't') {
-      models.wv.events.on("startup", function() {
-        wvx.ui.sidebar.selectTab("events");
+    if (!state.e) return;
+    models.wv.events.on('startup', function() {
+      wvx.ui.sidebar.selectTab('events');
```

Comment:
I would get rid of reference to this global `wvx` object in this file and pass a ui parameter in from `wv.main.js`